.github/workflows/pr.yml

```diff
@@ -29,28 +29,35 @@ jobs:
       - name: Compare current branch vs main branch
         # Run ansible-lint on this branch then compare vs main branch
         run: |
+          set +e
           set -x
           # fix randomness
           export PYTHONHASHSEED=42
           git checkout -b branch
           git fetch --unshallow origin main
-          cmd="ansible-lint --parseable --nocolor --profile=shared"
+          cmd="ansible-lint --parseable --force-color --profile=shared"
           # don't want to annotate through GHA
           unset GITHUB_ACTIONS
-          $cmd > branch.output || echo "Branch linting output written"
+          $cmd | tee branch.output
           git checkout main
-          $cmd > main.output || echo "Main linting output written"
+          $cmd > main.output
```

Comment:
No, you *need* to catch the potential exit 1 here, my previous comment was confusing, keep it as `$cmd > main.output || echo "Ran on main branch"` or something like that, but *only* for main branch, otherwise this will fail

Reply:
I added `set +e` at the beginning so no need for this code anymore.